.github/workflows/analyze.yml

```diff
@@ -3,8 +3,24 @@ name: Analyze
 on: push
 
 jobs:
+    Check_pr:
+        runs-on: ubuntu-latest
+        outputs:
+            is_pr: ${{ steps.check.outputs.is_pr }}
+        steps:
+            - name: Check if PR
+              id: check
+              run: |
+                  if [[ "${{ github.event.pull_request }}" != "" ]]; then
```

Comment:
Looks like `github.event.pull_request` is empty for this PR, which results in this statement evaluating to `False`. Maybe test it by checking what is the value stored in `github.event.pull_request`?